coastguard/coastguard.md

```diff
@@ -0,0 +1,138 @@
+# Submariner Multi-cluster network policy Support
+
+## Summary
```

Comment:
Thanks @shohamda for the PR. Sanjeev from RedHat is working on K8s Multi-cluster Network policy proposal, please take a look at it to see if you can collaborate your efforts - https://docs.google.com/document/d/1sLq_1CwfdvhmqxslQRdt7-RjLFsZRJkd/edit#heading=h.gjdgxs